### DISCUSSION TITLE
AuthAlreadyAssociated is raised regardless of social_user being disabled and SOCIAL_AUTH_RAISE_EXCEPTIONS set to False

### DISCUSSION THREAD

**Discussion by Goury** [python-social-auth/social-core#1335] [Q&A]
So I have this in my settings.py:

```
SOCIAL_AUTH_RAISE_EXCEPTIONS = False
SOCIAL_AUTH_PIPELINE = (
	'social_core.pipeline.social_auth.social_details',
	'social_core.pipeline.social_auth.social_uid',
	'social_core.pipeline.social_auth.auth_allowed',
#	'social_core.pipeline.social_auth.social_user',
	'social_core.pipeline.user.get_username',
	'social_core.pipeline.user.create_user',
	'social_core.pipeline.social_auth.associate_user',
	'social_core.pipeline.social_auth.load_extra_data',
	'social_core.pipeline.user.user_details',
)
```

And it raises

```
venv/lib/python3.8/site-packages/social_core/pipeline/social_auth.py in social_user
23.            raise AuthAlreadyAssociated(backend, msg) 
```

What is the point of these setting?
What am I doing wrong?

Yes, I do want to associate one social account to more than one django account.

social-auth-app-django==4.0.0
social-auth-core==3.3.3

**Comment by nijel** [comment]
* Did you restart Django after changing the pipeline? Is this the `settings.py` being used?
* `SOCIAL_AUTH_RAISE_EXCEPTIONS` only works in `SocialAuthExceptionMiddleware`, perhaps you don't have that one?

**Reply by Goury** [reply]
That took more than five years.
I don't even work on that project anymore.

Yes, `settings.py` was used.
Yes, `SocialAuthExceptionMiddleware` was used too.

I think I've found a bug in it and had to use the version I've fixed myself instead.
It was very long time ago and my memory isn't that good.
And I am too lazy to read through the repo history.
